<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>공격하기</string>
     </attribute>
     <widget class="QPushButton" name="skill1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="skill2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>81</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="skill3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>81</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>가방</string>
     </attribute>
     <widget class="QPushButton" name="item_Btn1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>121</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>randombox.png</normaloff>randombox.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>200</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="item_Btn2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>121</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>randombox.png</normaloff>randombox.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>도망가기</string>
     </attribute>
     <widget class="QPushButton" name="run">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>도망가기</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="bettlepokemon">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>180</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bettlepokemon_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="name_level">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QProgressBar" name="progressBar_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
	font: 18pt &quot;.AppleSystemUIFont&quot;;
	font: 18pt &quot;.AppleSystemUIFont&quot;;
     border-radius: 5px;
	background-color: rgb(179, 177, 181);
	font: 14pt &quot;American Typewriter&quot;;
 }

 QProgressBar::chunk {
     background-color: rgb(252, 1, 7);
	border-radius: 5px;
 }</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="format">
      <string>HP %p</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
	font: 18pt &quot;.AppleSystemUIFont&quot;;
     border-radius: 5px;
	background-color: rgb(179, 177, 181);
	font: 14pt &quot;American Typewriter&quot;;
 }

 QProgressBar::chunk {
     background-color: rgb(0, 112, 255);
	border-radius: 5px;
 }</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="format">
      <string>HP %p</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
	font: 18pt &quot;.AppleSystemUIFont&quot;;
     border-radius: 5px;
	background-color: rgb(179, 177, 181);
	font: 14pt &quot;American Typewriter&quot;;
 }

 QProgressBar::chunk {
     background-color: rgb(98, 255, 87);
	border-radius: 5px;
 }</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="format">
      <string>HP %p</string>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>270</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
	font: 18pt &quot;.AppleSystemUIFont&quot;;
     border-radius: 5px;
	background-color: rgb(179, 177, 181);
	font: 14pt &quot;American Typewriter&quot;;
 }

 QProgressBar::chunk {
     background-color: rgb(0, 112, 255);
	border-radius: 5px;
 }</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>HP %p</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>230</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
	font: 18pt &quot;.AppleSystemUIFont&quot;;
	font: 18pt &quot;.AppleSystemUIFont&quot;;
     border-radius: 5px;
	background-color: rgb(179, 177, 181);
	font: 14pt &quot;American Typewriter&quot;;
 }

 QProgressBar::chunk {
     background-color: rgb(252, 1, 7);
	border-radius: 5px;
 }</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>HP %p</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
